bin/doc2json/scripts/setup_grobid.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 # put in your pdf2json directory here
-export PDF2JSON_HOME=$HOME/git/s2orc-pdf2json
-
+export PDF2JSON_HOME=$(cd "$(dirname "$0")";cd ..;pwd)
```

Comment:
what's the logic here?

Reply:
I meant to get the absolute path of `bin/doc2json`, but mistook the current work path so it seems I did it in a stupid way:(
Have changed it to '$(pwd)/bin/doc2json` .